tests/src/JIT/Performance/CodeQuality/Benchstones/BenchF/BenchMk2/BenchMk2.cs

```diff
@@ -41,6 +41,27 @@ private static bool Bench()
         return true;
     }
 
+    private static bool VerifyResult()
+    {
+        const float DIFF = 0.0000001f;
```

Comment:
Diffing floats is hard because the epsilon between two values varies depending on the input. This epsilon (or delta between `x` and the next/previous value) is computed as `var epsilonToNext = Math.BitIncrement(x) - x` and `var epsilonToPrevious = x - Math.BitDecrement(x)`. -- Generally these two epsilon are in sync; but powers of 2 have them differ as do special values (like `MaxValue`/`MinValue`).

Generally, if defining a `diff`. It would be good to indicate (in an attached comment) the "exact mathematical result" expected, the "nearest representable floating-point result" to that value, and then the quantifying the diff based on that.

As is, this `diff` will be treated as `0` for large results and will be treated as very large for small results and there is no context as to what the actual result should be and how the diff correlates to that.

Reply:
Good point, thank you. I've improved it for the bisect and dmath classes but I'm not sure how to handle it for the two BenchMk classes (see the comment below).